README.md

```diff
@@ -2,6 +2,8 @@
 
 This module provides necessary services for running Bahmni
 
+This module also contains global property **bahmni.lookupExternalTerminologyServer** to enable/ disable concept search through [FHIR Terminology Service Module](https://github.com/Bahmni/openmrs-module-snomed)
```

Comment:
I would rather put the link to Bahmni SNOMED Wiki page instead of the Omod. A lot of things have to be correctly configured before this feature flag can be used.